submissions/kuslosolodka/html-css-popup/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="popup-button">
+      <label for="popup-btn">
+        <img class="popup-icon"
+            src="assets/icons/popup-button.png"
+            alt="popup icon">
+          </label>
+    </div>
+    <input type="checkbox" id="popup-btn" />
+    <div class="popup-menu">
+      <div class="icons">
+        <a href="#" tabindex="0"
+          ><img
+            src="assets/icons/calendar.png"
+            alt="calendar icon"
+          />Calendar</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/mail.png" alt="mail icon" />Mail</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/news.png" alt="news icon" />News</a
```

Comment:
Fix aspect ratio of images. [Read about this](https://jakearchibald.com/2022/img-aspect-ratio/)